### PR TITLE
V.12multiqc v1.12

### PR DESCRIPTION
**Pull request by carmelinacharalambous**
* updated to reflect develop branch and not main branch (removing previous PR)
## commit includes: ##

* multiqc v1.12 that is compatible with mosdepth output
* added contributor and creator info in *cwl
* added multiqc v.1.12 and delly v1.0.3 to summary.md
* in summary.md delly 0.9.1 documentation was listed as delly_call_0.9.1.md but file was named README.md changed file name

### CODE REVIEW

**Review by rhshah — APPROVED** [review]

**Comment by rhshah** [comment]
@carmelinacharalambous  I fixed some of the confusing doc issues hopefully going forward it won't be confusing!

File: multiqc/1.12/multiqc_1.12.cwl [hunk 1]
```diff
@@ -0,0 +1,84 @@
+class: CommandLineTool
+cwlVersion: v1.0
+$namespaces:
+  sbg: 'https://www.sevenbridges.com/'
```
Comment (rhshah) [inline multiqc/1.12/multiqc_1.12.cwl:4]: We need to add additional namespace for the tags we used below to give author information

**Review by rhshah — CHANGES_REQUESTED** [review]
Please change the cwl to have additional namespace

File: multiqc/1.12/multiqc_1.12.cwl (hunk 1 above)
Comment (carmelinacharalambous) [reply multiqc/1.12/multiqc_1.12.cwl:7]: done

**Review by rhshah — APPROVED** [review]

**Pull request merged**